crates/biome_js_analyze/src/analyzers/nursery/no_restricted_imports.rs

```diff
@@ -0,0 +1,77 @@
+use biome_analyze::context::RuleContext;
+use biome_analyze::{declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_deserialize_macros::Deserializable;
+use biome_js_syntax::inner_string_text;
+use biome_rowan::TextRange;
+use serde::{Deserialize, Serialize};
+
+use super::use_nodejs_import_protocol::AnyJsImportLike;
+
+declare_rule! {
+    /// Disallow specified modules when loaded by import or require.
+    ///
+    /// https://eslint.org/docs/latest/rules/no-restricted-imports
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```json
+    /// {
+    ///     "noRestrictedImports": {
+    ///         "options": {
+    ///             "paths": ["lodash"]
+    ///         }
+    ///     }
+    /// }
+    /// ```
+    ///
+    /// ```js
+    /// const lodash = require("lodash");
+    /// ```
```

Comment:
Unfortunately, we don't support setting options in example. We could just document the option. Any idea @ematipico ?

Reply:
It could be an excellent addition and maybe a good first issue. As for now, we only document the options.